kdump/include/runtest.sh

```diff
@@ -511,7 +506,82 @@ AppendConfig()
         shift
     done
 
-    rstrnt-report-log -l "${KDUMP_CONFIG}"
+    RhtsSubmit "${KDUMP_CONFIG}"
+}
+
+
+AppendSysconfig()
+{
+    Log "- Modifying /etc/sysconfig/kdump"
+
+    local KEY=$1
+    local ACTION=$2
+    local VALUE1=$3
+    local VALUE2=$4
+
+    if [ -z "$KEY" ] || [ -z "$ACTION" ] || [ -z "$VALUE1" ]; then
+        Error "- Missing KEY or ACTION or VALUE1."
+        return 1
+    elif [ "$ACTION" = "replace" ] && [ -z "$VALUE2" ]; then
+        Error "- Missing new_value for replacing."
+        return 1
+    elif ! grep "^$KEY=\"" "${KDUMP_SYS_CONFIG}"; then
+        Error "- Invalid KEY: $KEY."
+        return 1
+    fi
+
+    local kdump_sys_config_tmp="${KDUMP_SYS_CONFIG}.tmp"
+    \cp "${KDUMP_SYS_CONFIG}" "${kdump_sys_config_tmp}"
```

Comment:
What does the `\` mean, here?

Reply:
This is to overwrite the alias in case "cp -i" is set. Copied it from the original kdump test. May not be necessary in most cases, but no harm either.

Reply:
Ah, nice, thanks! I never knew.